lib/ui/profile/FarmDetails.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:farmsmart_flutter/ui/profile/FarmDetailsListItem.dart';
+import 'package:farmsmart_flutter/ui/common/ContextualAppBar.dart';
+import 'package:farmsmart_flutter/ui/common/ActionSheet.dart';
+import 'package:farmsmart_flutter/ui/common/ActionSheetListItem.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class _Strings {
+  static final String farmDetailsTitle = "Your Farm Details";
+  static final String actionSheetButtonTitle = "Cancel";
+  static final String actionSheetEdit = "Edit Profile";
+  static final String actionSheetDelete = "Delete Profile";
+}
+
+class _Constants {
+  static final EdgeInsets floatingButtonEdgePadding = const EdgeInsets.all(32);
+  static final EdgeInsets headerEdgePadding =
+      const EdgeInsets.only(left: 32, right: 32, top: 10.5, bottom: 16);
+
+  static final double bottomMargin = 112;
+  static final double buttonHeight = 48;
+
+  static final BorderRadius buttonRadius = BorderRadius.all(
+    Radius.circular(12),
+  );
+}
+
+class FarmDetailsViewModel {
+  List<FarmDetailsListItemViewModel> items;
+  final String buttonTitle;
+  final Function onConfirmDetails;
```

Comment:
```suggestion
  final Function continue;
```

Reply:
Flutter tells me that continue its a keyword

Reply:
LOL yes it is.. continueAction maybe?

Reply:
infact just "confirm" will do it

Reply:
Done